src/completers/completeReducer/index.js

```diff
@@ -0,0 +1,27 @@
+import onLoading from '../../effects/onLoading';
+import onSuccess from '../../effects/onSuccess';
+import onFailure from '../../effects/onFailure';
+import isStringArray from '../validate';
+
+// Given a reducer description, it returns a reducerHandler with all success and failure cases
+function completeReducer(reducerDescription) {
+  if (!reducerDescription || !reducerDescription.primaryActions) {
+    throw new Error('Reducer description is incomplete, should contain at least a primaryActions field');
+  }
+  if (!isStringArray(reducerDescription.primaryActions)) {
+    throw new Error('Primary actions must be a string array');
+  }
+  let reducerHandler = {};
+  reducerDescription.primaryActions.forEach(actionName => {
+    reducerHandler[actionName] = onLoading();
```

Comment:
we should be able to handle selectors to `onLoading`, `onSuccess`, `onFailure`.
What do you think?

Reply:
Mmmm I'm not completely sure about the comment above. Maybe if the developers want to handle selectors for these effects, they should use the `override` param